src/components/Todo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="todo">
+    <template class="todo-intarface">
+      <h1
+        :class="{
+          red: checkCountOfDoneTask() < 0.5,
+          yellow: checkCountOfDoneTask() >= 0.5 && checkCountOfDoneTask() < 1,
+          green: checkCountOfDoneTask() === 1,
+        }"
+      >
+        TODO LIST
+      </h1>
+      <input
+        class="todo-input"
+        v-on:keydown.enter="addTodo"
+        type="text"
+        v-model="todo"
+      />
+      <button class="todo-btn" @click="addTodo">add new task</button>
+    </template>
+    <ul class="todo-list">
+      <li
+        v-for="(t, idx) of todos"
+        :key="idx"
+        class="todo-item"
+        @click.self="removeTodo(idx)"
+      >
+        <input
+          class="todo-checkbox"
+          @change.self="changeStatus(t)"
+          type="checkbox"
+          name="is-done"
+        />
+        <span :class="{ done: t.status }" @click="removeTodo(idx)"
+          >{{ `${idx + 1}.` }} {{ t.title }}</span
+        >
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ToDo",
+  data() {
+    return {
+      todos: [],
+      todo: "",
+    };
+  },
+  methods: {
+    addTodo() {
+      if (this.todo) {
+        this.todos.push({
+          title: this.todo,
+          status: false,
+        });
+      }
+      this.todo = "";
+    },
+    changeStatus(task) {
+      task.status = !task.status;
+    },
+    removeTodo(idx) {
+      this.todos = this.todos.filter((el, id) => id !== idx);
```

Comment:
This looks good, but again, we aren't limited in characters - **use full names for variables.**

Additionally, don't use names like `element`, `item` or `data` - these are magic words which don't mean anything.
If we're filtering todos in filter, then use `todo` as a variable name.